test.sh

```diff
@@ -57,9 +57,12 @@ start pytest
 pytest context/app
 end pytest
 
+start lint
+cd context
+npm run lint
+end lint
```

Comment:
```suggestion
cd -
end lint
```

Reply:
Moved the cds outside of the start/end blocks.